tests/playTests.sh

```diff
@@ -322,6 +323,15 @@ ZCAT=./zstdcat "$ZSTDGREP" 2>&1 "1234" tmp_grep_bad.zst && die "Should have fail
 ZCAT=./zstdcat "$ZSTDGREP" 2>&1 "1234" tmp_grep_bad.zst | grep "No such file or directory" || true
 rm -f tmp_grep*
 
+println "\n===> zstdless tests"
+rm -f tmp_less
+echo "1234" > tmp_less
+zstd -f tmp_less
+lines=$("$ZSTDLESS" 2>&1 tmp_less.zst | wc -l)
```

Comment:
why not a test of the form `[ "$($ZSTDLESS ...)" = "1234" ]`? The zstd-ed encoding likely doesn't have any newlines in it and so will probably report as one line as well.